doc/backups.rst

```diff
@@ -72,4 +72,64 @@ To secure the user, use an extemely restricted permission in ~/.ssh/authorized_k
 Restoring
 ---------
 
-TODO
+How to backup production servers:
+=================================
+Our backup process consists of two steps:
+1- Our current backup script backs up the production instance in the borg_backups repository located in the project path.
+
+2- Afterwards, this borg repository is copied to an ftp server associated to each production instance.
+
+This FTP server is a free service provided by OVH for each dedicated server and must be manually instanciated via the OVH manager.
+
+TODO: Automate enabling the ftp backup server via OVH APIs.
+
+In order to manually backup a production instance, launch the following::
+
+    fab -c {RC_FILE} execute_backup_borg_repository
+
+In order to set a crontab for a production instance, use this command::
+
+    fab -c {RC_FILE} cron_backup_borg_repository
+
+
+These two commands use the following pattern:
+1. Make sure borg is installed.
+2. Put a configuration file on the remote host with the server FTP information.
+3. Creates a backup script from template with the appropriate remote host information and puts it on the remote host.
+4. For the second host, it sets a cron job to automatically backup the instance daily.
+
+
+How to list backups on a production server:
+===========================================
+Use the command::
+
+    fab -c {RC_FILE} list_backups
+
+How to restore a backup on a production instance:
+=================================================
+Use the command::
+
+    fab -c {RC_FILE} fetch_backup
+
+This command will fetch the backup from the ftp server. If you don't fix the name,
+by default it will get the last backup folder.
+
+You will have the backup assembl in a newly created `home` folder.
+
+The command `fetch_backup` will also move the backup up assembl to a newly created
+folder: assembl_backup.
+
+Run the command `create_backup_rc` in order to get the RC file creating the last configurations for this server. Keep a copy of this RC file on your localhost as well.
+The resulting file will be .local.rc. This will be used to restore the backup.
+
+If you want to launch this backup on a new machine, you should just `bootstrap_from_backup` using the backup RC file.
+
+If you want to restore the backup on the server in the newly created `assembl_backup` folder, you have to change _projectpath variable in the RC file to `/home/assembl_user/assembl_backup/assembl/`
+
+If you want this to be your new assembl instance, you have to shut down all supervisor
+processes in the usual directory.
+
+If there is a problem with the database restore on the new folder, youb light have to do
+`service postgresql restart` to shut down all processes if you ge the error stating that there are still other processes using the database.
+
+Since you changed the project path you will have to change the paths in the nginx configuration. Then you should restart nginx.
```

Comment:
explain how to generate the nginx file. Editing should be discouraged.

Reply:
done.